src/demos/popover.html

```diff
@@ -103,13 +103,38 @@ <h1 style="margin: 0 auto; text-align: center">Popover</h1>
 
       <!-- with close button -->
       <calcite-popover
+        style="max-width: 320px"
         label="auto placement popover"
         reference-element="popover-button-close"
         placement="auto"
         id="popover-close"
         dismissible
       >
-        <div style="padding: 12px 16px">I am a popover with a close button!</div>
+        <style>
+          .info {
+            padding: 10px;
+          }
+
+          .header {
+            font-weight: bold;
+          }
+        </style>
+        <div class="info">
```

Comment:
Can this be converted into a nested-popover story?

Reply:
Yeah good idea. I'll add one.